android/app/src/main/java/org/fossasia/openevent/core/track/session/TrackSessionsActivityViewModel.java

```diff
@@ -120,4 +123,16 @@ protected void onCleared() {
         super.onCleared();
         clearChangeListeners();
     }
+
+    public int getListPosition() {
+        return listPosition;
+    }
+
+    public void setListPosition(int listPosition) {
+        this.listPosition = listPosition;
+    }
+
+    public Intent openSessionDetail(Session session, Intent intent) {
+        return Views.openSessionDetails(session, intent);
+    }
```

Comment:
I don't think there is any need of storing and retrieving just listPosition here. And listPosition is not stable. It can change if items are sorted or filtered. Also, there is no need to put openSessionDetail in ViewModel, it can be in the View itself